pom.xml

```diff
@@ -1064,15 +1063,29 @@ under the License.
 			</build>
 		</profile>
 
+		<profile>
+			<id>java11-target</id>
```

Comment:
```suggestion
			<id>java11-target</id>
			<properties>
				<target.java.version>11</target.java.version>
			</properties>
```

Reply:
looks like I missed one thing for local testing 
```
./mvnw clean install -DskipTests -Dscala-2.12 -Pfast -Pskip-webui-build -U -T4 -Pjava11 -Pjava11-target
```
this line was working since it has `-Dfast` which turns off enforcer plugin...
Without that it still fails.
Need to make java version in enforcer plugin configurable to make it working just as 
```
./mvnw clean install -Pjava11-target
```